test/NuGet.Core.FuncTests/NuGet.Packaging.FuncTest/SigningTests/TimestampTests.cs

```diff
@@ -81,17 +81,11 @@ public async Task Timestamp_Verify_WithOfflineRevocation_ReturnsCorrectFlagsAndL
                     result.HasFlag(SignatureVerificationStatusFlags.UnknownRevocation).Should().BeTrue();
 
                     var errors = logs.Where(l => l.Level == LogLevel.Error);
-                    errors.Count().Should().Be(RuntimeEnvironmentHelper.IsWindows ? 2 : 1);
-
-                    if (RuntimeEnvironmentHelper.IsWindows)
-                    {
-                        errors.Should().Contain(w => w.Code == NuGetLogCode.NU3028 && w.Message.Contains("The revocation function was unable to check revocation because the revocation server could not be reached."));
-                        errors.Should().Contain(w => w.Code == NuGetLogCode.NU3028 && w.Message.Contains("The revocation function was unable to check revocation for the certificate."));
-                    }
-                    else
-                    {
-                        errors.Should().Contain(w => w.Code == NuGetLogCode.NU3028 && w.Message.Contains("unable to get certificate CRL"));
-                    }
+
+                    errors.Count().Should().Be(RuntimeEnvironmentHelper.IsMacOSX ? 1 : 2);
+
+                    SigningTestUtility.AssertOfflineRevocation(errors, LogLevel.Error, NuGetLogCode.NU3028);
+                    SigningTestUtility.AssertRevocationStatusUnknown(errors, LogLevel.Error, NuGetLogCode.NU3028);
```

Comment:
Do we need to differentiate three platforms? The tests seem failed.

Reply:
haha, I messed up and typoed the message. The test should be passing now.